sale_template_operating_unit/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2020 Pavlov Media
```

Comment:
The PR looks good, let's update the headers on these files for 2021. Besides that I am ok approving this

Reply:
@osi-scampbell This has been updated and commits squashed.